api/src/main/java/org/pmiops/workbench/config/GenderRaceEthnicityConceptCache.java

```diff
@@ -0,0 +1,65 @@
+package org.pmiops.workbench.config;
+
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+import com.google.common.cache.LoadingCache;
+import org.pmiops.workbench.cdr.cache.GenderRaceEthnicityConcept;
+import org.pmiops.workbench.cdr.cache.GenderRaceEthnicityType;
+import org.pmiops.workbench.cdr.dao.ConceptDao;
+import org.pmiops.workbench.cdr.model.Concept;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.ScopedProxyMode;
+import org.springframework.web.context.annotation.RequestScope;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+@Configuration
+public class GenderRaceEthnicityConceptCache {
+
+    public static final String MAIN_CONFIG_ID = "main";
+
+    private static final Map<String, Class<?>> GENDER_RACE_ETHNICITY_MAP = new HashMap<>();
+
+    static {
+        GENDER_RACE_ETHNICITY_MAP.put(MAIN_CONFIG_ID, GenderRaceEthnicityConcept.class);
+    }
+
+    @Bean
+    @Qualifier("genderRaceEthnicityCache")
+    LoadingCache<String, Object> getGenderRaceEthnicityCache(ConceptDao conceptDao) {
+        // Cache configuration in memory for 24 hours.
+        return CacheBuilder.<String, GenderRaceEthnicityConcept>newBuilder()
+                .expireAfterWrite(10, TimeUnit.SECONDS)
```

Comment:
I think you could make this 1 hour or 1 day or whatever. This isn't going to need to change.

Reply:
Ooops... I was planning on changing it to 24 hours indicated by my comment, but forgot to change it back from 10 seconds which i used for testing that it was refreshing. Done.